developer/typical-scenarios/deprecate-a-pack.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: dev-doc
+title: How to deprecate a pack
+id: deprecate-a-pack
+---
+
+There are 2 flags available in pack which allows easy sunset of specific pack version
+~~~ruby
+ignore true|false
+enabled true|false
+~~~
+
+# ignore
+
+The flag is used before loading of the pack. If ignore flag is set to **true**, the pack will not be re-loaded (updated) in OneOps but it will not remove the current one (already loaded) so it will stay there dormant.
```

Comment:
use `ignore` and `true` (code) .. rather than bold and nothing. Also "If the `ignore` flag ..